src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java

```diff
@@ -221,7 +228,8 @@ private void removeSessionLevelPrivileges(InstructorAttributes instructorToEdit,
                     String valueForSectionName =
                             getRequestParamValue(Const.ParamsNames.INSTRUCTOR_SECTION_GROUP + i
                                                  + Const.ParamsNames.INSTRUCTOR_SECTION + j);
-                    if (isSectionGroupSpecial && valueForSectionName != null && sectionNamesTable.containsKey(valueForSectionName)) {
+                    if (isSectionGroupSpecial && valueForSectionName != null
+                            && sectionNamesTable.containsKey(valueForSectionName)) {
```

Comment:
Same as somewhere above:
the indented `&&` suggests only two conditionals when there are in fact three, maybe don't indent the second line